### ISSUE TITLE
Homebridge not transferring accessories

### ISSUE THREAD

**Issue by Pdub76** [bassrock/homebridge-flair#394]
Is anyone else not getting their accessories to transfer via Homebridge to HomeKit?
I have my API credentials, username and pw all setup.
Any guidance or assistance would be appreciated!

**Comment by VTDARKSIM** [comment]
Same here, though it’s only been a few hours and HomeKit has taken overnight to pull in new accessories before, so we’ll see about tomorrow.

Update next day: still not showing any Flair accessories in either HomeBridge or HomeKit.

**Comment by VTDARKSIM** [comment]
So this was working for months, and suddenly all Flair devices stopped updating in HomeKit no matter what I tried. I eventually nuked the plugin, reinstalled it, re-added the devices, moved them to their respective rooms and recreated the automations, and it worked. For about a month. All Flair devices will not update again as of this morning, though they are accessible in the Flair app.
Has anyone else had this issue?

**Comment by passionatelogic** [comment]
Yes! Same exact issue here. Flair vents stay working in my Home app for about a month and then I can't control them again unless I remove them and start over.
